src/utils.ts

```diff
@@ -137,7 +137,10 @@ function getReportHeaders(
 
     const storage = getAssemblyName(unittests)
 
-    const dllName = storage.split('/').pop()
+    const isWindows = /^[a-z]:\\/i.test(storage)
```

Comment:
May be worth using something like:
```
const isWindows = /^win/i.test(process.platform)
```

Reply:
ah I get it, its not about which OS its running on, its the path inside a `storage` attribute inside a trx file.